core/lib/common/worker.js

```diff
@@ -309,6 +341,8 @@ module.exports = class Worker {
 	}
 
 
+	// creates a tunnel a specified DUT port
+	// In the case of a virtual worker, can we map the qemu devices ports directly, without this???
 	async createTunneltoDUTSSH(
```

Comment:
@rcooke-warwick This might be easier with ssh2-promise https://github.com/sanketbajoria/ssh2-promise#connect-to-ssh-server-via-hopping

Reply:
@jakogut the only problem with the server hopping commands with the ssh2 library (as far as I can tell), is that it doesn't account for the `host <UUID>` option thats required to get into a device connected to balena cloud via the proxy.